packages/data-service-generator/src/tests/create-data-service-with-grpc.spec.ts

```diff
@@ -0,0 +1,67 @@
+import entities from "./entities";
+import roles from "./roles";
+import { AppInfo } from "@amplication/code-gen-types";
+import { appInfo, MODULE_EXTENSIONS_TO_SNAPSHOT } from "./appInfo";
+import { EnumResourceType } from "../models";
+import { createDataService } from "../create-data-service";
+import { MockedLogger } from "@amplication/util/logging/test-utils";
+import { join } from "path";
+import { AMPLICATION_MODULES } from "../generate-code";
+
+const newAppInfo: AppInfo = {
+  ...appInfo,
+  settings: {
+    ...appInfo.settings,
+    serverSettings: {
+      generateGraphQL: false,
+      generateRestApi: true,
+      serverPath: "",
+    },
+  },
+};
+
+jest.setTimeout(100000);
+
+describe("createDataService", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+  describe("when graphql is disabled", () => {
```

Comment:
Is "when graphql is disabled" the correct name here? (it is indeed disabled but I feel the test is about "when grpc is enabled")